docs/reference/edot-sdks/android/getting-started.md

```diff
@@ -24,7 +24,6 @@ Set up the Elastic Distribution of OpenTelemetry Android (EDOT Android) in your
 | Requirement                                       | Minimum version                                                                                           |
 |---------------------------------------------------|-----------------------------------------------------------------------------------------------------------|
 | [{{stack}}](https://www.elastic.co/elastic-stack) | 8.18                                                                                                      |
-| Android Gradle plugin                             | 7.4.0                                                                                                     |
```

Comment:
Addresses [this issue](https://github.com/elastic/apm-agent-android/issues/574#issuecomment-3096436295).